src/sign/Sign.service.ts

```diff
@@ -396,12 +396,12 @@ export class SignService {
       );
     }
 
-    if (!(user.roles.includes('Submitter') || user.roles.includes('Sponsor')) ) {
+    if (!(user.roles.includes('Submitter') || user.roles.includes('Sponsor') || user.roles.includes('Initial Authorizer')) ) {
       throw new EaseyException(
         new Error('This requires the Sponsor or Submitter role'),
```

Comment:
For consistency, this should match the error in the response:
```suggestion
        new Error('This requires the Sponsor or Submitter or Initial Authorizer role'),
```

Reply:
these changes are already been discussed

Reply:
Right, but looking at the linked issue, @mark-hayward-erg noted that there will be cases when the _only_ role returned is "Initial Authorizer". This issue is just for the API, so I think the ECMPS discussion can go elsewhere, but I do think the Error descriptions should probably match.

Reply:
Yes but this message is already been there and approved so this will remain the same

Reply:
If it's already approved I'm not sure why you requested me for a review? I'll go ahead and pass this back to @spetros-do.

Reply:
I got the point lemme do that.

Reply:
@shahbazkhan1999 @maxdiebold-erg The error message returned to the user needs to match the validation logic that is being applied.

The change here is to allow users that only have the Initial Authorizer to be able to submit, so the error message should read "This requires the Sponsor or Submitter or Initial Authorizer role".

Reply:
Alright @mark-hayward-erg